techmatters/flex-plugins
# Banner on created Case

## Description
Shows a banner when a case is created with:
- Text: Case # 1234 created and contact added
- Button: Cancel Case

TODO:
- ~Fix visual issues~ (done)
- Tests will come in a follow up PR

### Related Issues
Fixes https://tech-matters.atlassian.net/browse/CHI-2380

### Verification steps
- Fill a contact
- Create a new Case
- Verify the banner shows up
- Verify the cancel button works
